packages/cursorless-vscode/package.json

```diff
@@ -399,6 +411,198 @@
           },
           "additionalProperties": false
         },
+        "cursorless.scopeVisualizer.colors.dark": {
```

Comment:
Should we flatten these?  Might make it easier to change them

Reply:
I would recommend as many settings as possible to be changeable in the gui

Reply:
Hmm because these aren't proper color theme colors (because we need to access them for programmatic blending), they are a bit annoying to edit with settings gui:

<img width="761" alt="image" src="https://github.com/cursorless-dev/cursorless/assets/755842/73215cc1-6dc9-41bd-b373-2ed733ff5b62">

If I make them one nested object, then when you click the settings button it dumps all the defaults to json and shows you rendered color and color picker

<img width="742" alt="image" src="https://github.com/cursorless-dev/cursorless/assets/755842/28a626dc-ba15-4a7a-923e-4c4a531e4fef">

wdyt?  I don't feel super strongly either way, but it almost seems like ux is better without settings gui here